### PR TITLE
Feature/rowscols

### PR DESCRIPTION
**Pull request by lessthanoptimal**
Thank you for your contribution to the EJML project.

Before submitting this PR, please make sure:
- [x] Your code is covered by tests
- [x] You ran `./gradlew spotlessJavaApply`

### CODE REVIEW

**Comment by lessthanoptimal** [comment]
@JozsefKutas does this look good?

File: main/ejml-simple/src/org/ejml/simple/SimpleBase.java [hunk 1]
```diff
@@ -795,6 +795,41 @@ public void setRow( int row, int startColumn, double... values ) {
         ops.setRow(mat, row, startColumn, values);
     }
 
+    /**
+     * <p>Copies the vector into the specified row. The 'src' vector and be a row or column vector as long as it
+     * has the correct length.</p>
+     *
+     * @param row Row in 'this'
+     * @param src Vector which is to be copied into the row
```
Comment (JozsefKutas) [inline main/ejml-simple/src/org/ejml/simple/SimpleBase.java]:
```suggestion
     * <p>Copies the vector into the specified row. The 'src' vector can be a row or column vector as long as it
     * has the correct length.</p>
     *
     * @param row Row in 'this'
     * @param src Vector which is to be copied into the row
```

File: main/ejml-simple/src/org/ejml/simple/SimpleBase.java [hunk 2]
```diff
@@ -810,6 +845,53 @@ public void setColumn( int column, int startRow, double... values ) {
         ops.setColumn(mat, column, startRow, values);
     }
 
+    /**
+     * <p>Copies the vector into the specified column. The 'src' vector and be a row or column vector as long as it
+     * has the correct length.</p>
+     *
+     * @param column Column in 'this'
+     * @param src Vector which is to be copied into the row
```
Comment (JozsefKutas) [inline main/ejml-simple/src/org/ejml/simple/SimpleBase.java]:
```suggestion
     * <p>Copies the vector into the specified column. The 'src' vector can be a row or column vector as long as it
     * has the correct length.</p>
     *
     * @param column Column in 'this'
     * @param src Vector which is to be copied into the column
```

File: main/ejml-simple/src/org/ejml/simple/SimpleBase.java [hunk 3]
```diff
@@ -1073,6 +1157,28 @@ public T extractVector( boolean extractRow, int element ) {
...
+     * @param row Row in the matrix
+     * @return Extracted vector
+     * @see #extractVector(boolean, int)
+     */
+    public T getRow( int row ) {
+        return extractVector(true, row);
+    }
+
+    /**
+     * Returns the specified column in 'this' matrix as a row vector.
```
Comment (JozsefKutas) [inline main/ejml-simple/src/org/ejml/simple/SimpleBase.java]:
```suggestion
     * Returns the specified column in 'this' matrix as a column vector.
```

**Review by JozsefKutas — APPROVED** [review]
I've added a few suggestions for the javadoc. Otherwise it looks good to me.

**Comment by lessthanoptimal** [comment]
Thanks for all those JavaDoc edits. Github needs to have a less clunky way to accept them.

**Pull request merged**